sc-tools/sc-builder/src/builder.cpp

```diff
@@ -5,281 +5,133 @@
  */
 
 #include "builder.hpp"
-#include "keynodes.hpp"
 #include "scs_translator.hpp"
+#include "gwf_translator.hpp"
 
-#include <boost/filesystem.hpp>
 #include <memory>
-#include <boost/algorithm/string.hpp>
 
 #include <fstream>
-#include <unordered_set>
-
-namespace impl
-{
-std::unordered_set<std::string> gSupportedFormats = {"scs", "scsi"};
-
-std::string NormalizeExt(std::string ext)
-{
-  utils::StringUtils::ToLowerCase(ext);
-  return ext;
-}
-
-bool IsSupportedFormat(std::string const & fileExt)
-{
-  return gSupportedFormats.find(NormalizeExt(fileExt)) != gSupportedFormats.end();
-}
-
-class ExtParser
-{
-public:
-  void operator()(std::string const & file_path)
-  {
-    if (!file_path.empty())
-    {
-      std::ifstream infile(file_path);
-      std::string line;
-      while (std::getline(infile, line))
-      {
-        if (line.empty())
-          continue;
-        m_names.emplace_back(line);
-      }
-
-      m_params.clear();
-      m_params.reserve(m_names.size() + 1);
-      for (auto const & v : m_names)
-        m_params.push_back(v.c_str());
-      m_params.push_back(nullptr);
-    }
-
-    m_params.push_back(nullptr);
-  }
-
-  std::vector<const sc_char *> & GetParams()
-  {
-    return m_params;
-  }
-
-private:
-  std::vector<const sc_char *> m_params;
-  std::vector<std::string> m_names;
-};
-
-}  // namespace impl
 
 Builder::Builder() = default;
 
 bool Builder::Run(BuilderParams const & params, sc_memory_params const & memoryParams)
 {
   m_params = params;
 
-  CollectFiles();
-
   // initialize sc-memory
-  bool noErrors = true;
-  impl::ExtParser extParser;
-  extParser(m_params.m_enabledExtPath);
-
   ScMemory::Initialize(memoryParams);
+
+  ScRepoPathCollector::Sources excludedSources, checkSources;
```

Comment:
Accotding to the codestyle you should split it to the 2 lines

Reply:
?

Reply:
It's fixed